addons/escoria-core/game/esc_logger.gd

```diff
@@ -0,0 +1,193 @@
+class ESCLoggerBase:
+	# Perform emergency savegame 
+	signal perform_emergency_savegame
+	
+	# Log file format
+	const  LOG_FILE_FORMAT: String = "log_%s_%s.log"
+	
+	# Valid log levels
+	enum { LOG_ERROR, LOG_WARNING, LOG_INFO, LOG_DEBUG, LOG_TRACE }
+
+	# A map of log level names to log level ints
+	var _level_map: Dictionary = {
+		"ERROR": LOG_ERROR,
+		"WARNING": LOG_WARNING,
+		"INFO": LOG_INFO,
+		"DEBUG": LOG_DEBUG,
+		"TRACE": LOG_TRACE,
+	}
+	
+	# Configured log level
+	var _log_level: int
+
+
+	# Constructor
+	func _init():
+		_log_level = _level_map[ESCProjectSettingsManager.get_setting(
+			ESCProjectSettingsManager.LOG_LEVEL.to_upper()
+		)]
```

Comment:
```suggestion
		_log_level = _level_map[ESCProjectSettingsManager.get_setting(
			ESCProjectSettingsManager.LOG_LEVEL
		).to_upper()]
```

I think the idea here is to uppercase the value that the project setting returns for use in the log level dictionary.